lib/sqlalchemy/sql/selectable.py

```diff
@@ -6846,9 +6892,11 @@ def bindparams(
         self.element = self.element.bindparams(*binds, **bind_as_values)
         return self
 
```

Comment:
**mike bayer** (zzzeek) wrote:

remove

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/4325